src/modules/shared/layouts/AppLayout/AppLayout.tsx

```diff
@@ -60,6 +68,8 @@ import { scrollTo } from '@shared/utils/scroll-to-top';
 
 import packageJson from '../../../../../package.json';
```

Comment:
REQ: This is not part of your changes but does not seem to be correct. Please remove

Reply:
Turns out it is used to set the build version on the window object. We should keep this.